### PR TITLE
(PE-23484) Conditionally disable PDB

### PR DESCRIPTION
**Pull request by justinstoller**

### CODE REVIEW

**Comment by justinstoller** [comment]
Saw this issue with my recent PR. This is I *think* the approach to take. Still need to test it locally though.

File: acceptance/suites/tests/00_smoke/master_starts_if_agent_run_before_ssl_files_inited.rb [hunk 1]
```diff
@@ -7,6 +7,10 @@
 puppetservice=options['puppetservice']
 ssldir = master.puppet['ssldir']
 backup_ssldir = master.tmpdir("agent_run_before_master_init_ssldir_backup")
+reports_termini = on(master, puppet("config get --section master reports")).stdout.chomp
```
Comment (justinstoller) [inline acceptance/suites/tests/00_smoke/master_starts_if_agent_run_before_ssl_files_inited.rb]: `config has no 'get' subcommand` (this should be `print` I think)

**Comment by justinstoller** [comment]
I've tested this in FOSS with redhat (pdb integration) and ubuntu (no pdb), testing in pe-pse next.

**Comment by justinstoller** [comment]
I've tested this from the irving branch of pe-pse as well using both redhat7 and ubuntu1604 masters without issues.

File: acceptance/suites/tests/00_smoke/master_starts_if_agent_run_before_ssl_files_inited.rb [hunk 2]
```diff
@@ -7,6 +7,10 @@
 puppetservice=options['puppetservice']
 ssldir = master.puppet['ssldir']
 backup_ssldir = master.tmpdir("agent_run_before_master_init_ssldir_backup")
+reports_termini = on(master, puppet("config print --section master reports")).stdout.chomp
+puppetdb_enabled = reports_termini =~ /puppetdb/
+route_file_exists = on(master, "ls /etc/puppetlabs/puppet/").stdout =~ /routes\.yaml/
```
Comment (haus) [inline acceptance/suites/tests/00_smoke/master_starts_if_agent_run_before_ssl_files_inited.rb]: nit: I think `test -f /etc/puppetlabs/puppet/routes.yaml` would be better here, but :woman_shrugging:

**Comment by haus** [comment]
I think this looks reasonable, but I have no idea what problem it is trying to solve (I assume some configuration difference between PE tests and FOSS tests?) Maybe a few comments in the disable/re-enable code to explain what is happening and/or a more detailed commit message? Otherwise I'm good.

**Comment by Magisus** [comment]
Thanks for those comments.

**Pull request merged**